crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1142,6 +1142,28 @@ impl World {
         self.archetypes.clear_entities();
         self.entities.clear();
     }
+
+    /// Create a version of this [`World`] which can be sent to another thread
+    ///
+    /// # Panics
+    ///
+    /// If `self` contains any [`!Send`] resources, e.g. from calls to [`World::insert_non_send`]
+    ///
+    /// [`!Send`]: Send
+
+    pub fn turtle(self) -> Turtle {
```

Comment:
Hmm. I wonder if this method or a `From` impl is better...

Reply:
It has the potential to be a relatively expensive operation, so I don't want it to be so implicit.